conftest.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from enum import Enum
+from supplier import models
+
+
+class TestFields(Enum):
+    USER_NAME_TEST = 'ScrappyKoko'
+    FIRST_NAME_TEST = 'Zohan'
+    LAST_NAME_TEST = 'Dvir'
+    PASSWORD_TEST = 'silkysmooth'
+    USER_TYPE_TEST = 'supplier'
+    BUSINESS_NAME_TEST = 'hair_salon_inc'
+
+
+class Fields(Enum):
+    user_name = 1
+    first_name = 2
+    last_name = 3
+    password = 4
+    user_type = 5
+    business_name = 6
+
+
+@pytest.fixture
+def supplier_test():
```

Comment:
This does not need to have "test" in it. it is very confusing when later on, in the test itself, one of the variables is called "supplier_test".